source/packages/modulo_core/include/modulo_core/Exceptions/NotImplementedExcpetion.hpp

```diff
@@ -1,23 +1,11 @@
-/**
- * @class NotImplementedException
- * @brief Exception to raise when a functionnality is not implemented
- * @author Baptiste Busch
- * @date 2019/09/11
- */
 #pragma once
 
-#include <iostream>
 #include <exception>
+#include <iostream>
 
-namespace Modulo
-{
-	namespace Exceptions
-	{
-		class NotImplementedException : public std::logic_error
-		{
-		public:
-			explicit NotImplementedException(const std::string& msg) : logic_error(msg)
-			{};
-		};
-	}
-}
+namespace modulo::core::exceptions {
```

Comment:
Getting rid of these nested namespaces is so satisfying :smile:

Reply:
ah ah yes indeed!